changelog/1403.md

```diff
@@ -0,0 +1,4 @@
+# level: patch
+# reference: issue 1403
```

Comment:
The error from `yarn changelog` is because these have leading `#` on them.  But I think this change does not need a changelog entry -- we only make such entries for changes that users would need to know about when upgrading (such as something that might break).  So I think it's best to remove this file.  But good thinking adding it!

Reply:
Thank you for your feedback! I will make those changes.